src/main/java/mat/client/advancedsearch/AdvancedSearchPanel.java

```diff
@@ -169,18 +167,16 @@ private void buildCommonSearchHP(HorizontalPanel advancedSearchRow) {
 	}
 
 	private void buildModelType() {
-		if (!isMeasure) {
-			modelTypeListBox = new ListBox();
-			modelTypeListBox.getElement().setClassName("navPill");
-			modelTypeListBox.setWidth("200px");
-			modelTypeListBox.getElement().getStyle().setMarginRight(15, Style.Unit.PX);
-			modelTypeListBox.getElement().getStyle().setMarginLeft(0, Style.Unit.PX);
-			modelTypeListBox.setId("modelType");
-			modelTypeListBox.setHeight(HEIGHT_OF_BOXES);
-			modelTypeListBox.addItem("Model Type: All", SearchModel.ModelType.ALL.toString());
-			modelTypeListBox.addItem("Model Type: FHIR / CQL Only", SearchModel.ModelType.FHIR.toString());
-			modelTypeListBox.addItem("Model Type: QDM / CQL Only", SearchModel.ModelType.QDM.toString());
-		}
+		modelTypeListBox = new ListBox();
+		modelTypeListBox.getElement().setClassName("navPill");
+		modelTypeListBox.setWidth("200px");
+		modelTypeListBox.getElement().getStyle().setMarginRight(15, Style.Unit.PX);
+		modelTypeListBox.getElement().getStyle().setMarginLeft(0, Style.Unit.PX);
+		modelTypeListBox.setId("modelType");
+		modelTypeListBox.setHeight(HEIGHT_OF_BOXES);
+		modelTypeListBox.addItem("Model Type: All", SearchModel.ModelType.ALL.toString());
+		modelTypeListBox.addItem("Model Type: FHIR / CQL Only", SearchModel.ModelType.FHIR.toString());
+		modelTypeListBox.addItem("Model Type: QDM / CQL Only", SearchModel.ModelType.QDM.toString());
```

Comment:
is QDM/QDM not needed?

Reply:
Missed that adding it in.

Reply:
Just pushed the fix for this. Also had to change the CQLLibraryDAOImpl a bit because ModelType had to change (couldn't use QDM).